x-pack/plugins/file_upload/public/components/geojson_upload_form/index_name_form.tsx

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { ChangeEvent, Component } from 'react';
+import { EuiFormRow, EuiFieldText, EuiCallOut, EuiSpacer } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { validateIndexName } from '../../util/indexing_service';
+
+export interface Props {
+  indexName?: string;
+  indexNameError?: string;
+  onIndexNameChange: (name: string, error?: string) => void;
+}
+
+interface State {
```

Comment:
Can you remove state since existingIndexNames is not longer used, nothing is stored in state